### PR TITLE
[IMP] inventory: update 2 step routes doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to the `receipts_delivery_two_steps` doc in Odoo's Inventory documentation, specific to 17.0.

Here's what I did:
- Retooled some of the instructions/flow for 17.2 update that affected all multi-step routes
- Replaced all screenshots for 17.2 UI, removed some screenshots no longer relevant

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

[Project Task Link](https://www.odoo.com/web#id=3878113&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @odoo/inventory-doc-review, this PR is ready for review when one of you has a chance. All that needs to be reviewed are the few lines that were changed, and the replaced screenshots. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 1]
```diff
@@ -81,56 +81,46 @@ quote as a new purchase order.
    name.
 
 After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button will appear in
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:83]:
```suggestion
After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button appears at
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 2]
```diff
@@ -81,56 +81,46 @@ quote as a new purchase order.
...
-happen until the receipt is completed. The status of the internal transfer will only change to
-:guilabel:`Ready` once the receipt has been marked as :guilabel:`Done`.
+Once the purchase order is confirmed, a receipt is generated and ready to process.
+
+Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Receipts` task card on the
+:guilabel:`Inventory Overview` dashboard. This reveals a list of all receipts to process.
 
 Click on the :guilabel:`Receipt` associated with the purchase order, then click :guilabel:`Validate`
-to complete the receipt and move the product to the :guilabel:`Input Location`.
+to complete the receipt and move the product to the :guilabel:`WH/Input`.
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:100]:
```suggestion
to complete the receipt and move the product to :guilabel:`WH/Input`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 3]
```diff
@@ -81,56 +81,46 @@ quote as a new purchase order.
...
 
-Click on the :guilabel:`Transfer` associated with the purchase order, then click
-:guilabel:`Validate` to complete the receipt and move the product to stock. Once the transfer is
-validated, the product enters the stock and is available for customer deliveries or manufacturing
-orders.
+Click :guilabel:`Validate` to complete the transfer and move the product from :guilabel:`WH/Input`
+to :guilabel:`WH/Stock`.
 
-.. image:: receipts_delivery_two_steps/two-step-validate-transfer.png
+Once the transfer is validated, the product enters the stock and is available for customer
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:120]:
```suggestion
Once the transfer is validated, the product enters inventory and is available for customer
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 4]
```diff
@@ -148,72 +138,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
...
-   The delivery order associated with the :abbr:`SO (Sales Order)` can also be quickly accessed by
-   clicking on the :guilabel:`Delivery` smart button again, and choosing the delivery order on the
-   :guilabel:`Transfers` page (which should now be marked as :guilabel:`Ready`).
+Navigate back to the original sales order by going to :menuselection:`Sales app`, and selecting the
+associated sales order.
 
-.. image:: receipts_delivery_two_steps/deliver-two-step-kanban.png
-   :align: center
-   :alt: The delivery order can be seen in the Inventory Kanban view.
+The :guilabel:`Delivery` smart button now indicates there are 2 transfers. Click the
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:180]:
```suggestion
The :guilabel:`Delivery` smart button now indicates there are two transfers. Click the
```

**Review by hojo-odoo — APPROVED** [review]
Hey @brse-odoo,

Nice job with these updates. I'm approving with a few small suggestions.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 1 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:83]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 2 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:100]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 3 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:120]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 4 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:180]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR is ready for final content review. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 5]
```diff
@@ -39,7 +39,7 @@ setting, :guilabel:`Save` the changes.
    Activating the :guilabel:`Multi-Step Routes` setting will also activate the :guilabel:`Storage
    Locations` feature.
 
-.. image:: receipts_delivery_two_steps/multi-step-routes.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-settings.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:42]: Not against it...just don't see the change here other than adding 10kb.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 6]
```diff
@@ -80,57 +80,46 @@ quote as a new purchase order.
...
+
+Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Receipts` task card on the
+:guilabel:`Inventory Overview` dashboard. This reveals a list of all receipts to process.
 
 Click on the :guilabel:`Receipt` associated with the purchase order, then click :guilabel:`Validate`
-to complete the receipt and move the product to the :guilabel:`Input Location`.
+to complete the receipt and move the product to :guilabel:`WH/Input`.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-receipt.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-receipt-form.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:101]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 7]
```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
 the top, above the :abbr:`SO (Sales Order)` form. Click the :guilabel:`Delivery` smart button to
 reveal the associated receipt.
 
-.. image:: receipts_delivery_two_steps/two-step-sales-quote.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:140]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 8]
```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
...
+:guilabel:`Delivery` smart button to reveal each transfer tied to this sales order: the completed
+picking, and the new delivery (WH/OUT) operation. Click the delivery operation to open the delivery
+order.
 
-Click on the delivery order associated with the :abbr:`SO (Sales Order)`, then click on
-:guilabel:`Validate` to complete the move.
+Once ready, click :guilabel:`Validate` to complete the delivery.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-delivery.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-order.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:186]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 9]
```diff
@@ -80,57 +80,46 @@ quote as a new purchase order.
...
-validated, the product enters the stock and is available for customer deliveries or manufacturing
-orders.
+Click :guilabel:`Validate` to complete the transfer and move the product from :guilabel:`WH/Input`
+to :guilabel:`WH/Stock`.
 
-.. image:: receipts_delivery_two_steps/two-step-validate-transfer.png
+Once the transfer is validated, the product enters inventory and is available for customer
+deliveries or manufacturing orders.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-internal-transfer.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:122]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 10]
```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
...
 
-.. image:: receipts_delivery_two_steps/validate-two-step-pick.png
+Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
+automatically reserves the product. Quantities can also be manually set by changing the value in the
+:guilabel:`Quantity` column to match the value in the :guilabel:`Demand` column.
+
+Once ready, click :guilabel:`Validate` to complete the picking and move the product from
+:guilabel:`WH/Stock` to :guilabel:`WH/Output.`
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:167]: Too zoomed in, would include the larger form/UI

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Approving with similar comments to the previous PRs.
Can pass to SALI when done, thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 6 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:101]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 7 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:140]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 8 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:186]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 10 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:167]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst (hunk 5 above)
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:42]: The old screenshot was in a 15.0 db, and was replaced with a screenshot from a 17.2 database. It does look similar, but there are very small differences - the text is slightly thinner and less bold

**Comment by brse-odoo** [comment]
> Hi @brse-odoo Approving with similar comments to the previous PRs. Can pass to SALI when done, thanks!

Hi @samueljlieber, this PR has passed us-doc-review and is ready for your final lookover before merge, thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst [hunk 11]
```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
...
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
    :align: center
-   :alt: After confirming the sales order, the Delivery smart button appears showing two items
-         associated with it.
+   :alt: Delivery smart button on sales order form.
 
 .. tip::
    Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
    :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
    :guilabel:`Pick` kanban card.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:147]: Small fix :)
```suggestion
   :guilabel:`Pick` Kanban card.
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo, nice job on the improvements to this doc! Your changes look good to me, approving and delegating to you with one quick fix.
..
@robodoo delegate=brse-odoo

**Comment by brse-odoo** [comment]
> Hi @brse-odoo, nice job on the improvements to this doc! Your changes look good to me, approving and delegating to you with one quick fix. .. @robodoo delegate=brse-odoo

@robodoo r+

**Pull request closed without merging**